app/controllers/api/migrations.php

```diff
@@ -580,7 +580,7 @@
 App::get('/v1/migrations/firebase/connect')
     ->desc('Authorize with firebase')
     ->groups(['api', 'migrations'])
-    ->label('origin', '*')
+    ->label('scope', 'migrations.read')
```

Comment:
hmm shouldn't this be a write scope as there's an updateDocument happening here?

Reply:
Makes senses, PR to update it [here](https://github.com/appwrite/appwrite/pull/6083)